org.tenacityaudio.Tenacity.yaml

```diff
@@ -0,0 +1,272 @@
+app-id: org.tenacityaudio.Tenacity
+default-branch: stable
+# freedesktop-sdk should work too, but causes problems with
+# the timeline mouse cursor on some systems for unkown reasons
+runtime: org.gnome.Platform
+runtime-version: '40'
+sdk: org.gnome.Sdk
+command: tenacity
+rename-desktop-file: tenacity.desktop
+rename-icon: tenacity
+rename-appdata-file: tenacity.metainfo.xml
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --share=network # Tenacity uses network sockets for IPC
+  - --socket=pulseaudio
+  - --device=all # ALSA
```

Comment:
I wonder is it necessary to have both `--socket=pulseaudio` and `--device=all` here?

I suppose if Tenacity wants to support PipeWire, PulseAudio and pure ALSA users out of the box we need all three (the third being the PipeWire arg below). But is that a goal here? Would some Tenacity functionality be considerably better using PipeWire (i.e. will Tenacity recommend people use it?)

(If I sent this twice that was my bad, I clicked "start a review" instead of "add single comment"...)

Reply:
> I wonder is it necessary to have both `--socket=pulseaudio` and `--device=all` here?

Socket and devices are not the same thing.

Reply:
> But is that a goal here?

Yes.

> Would some Tenacity functionality be considerably better using PipeWire (i.e. will Tenacity recommend people use it?)

Yes, but not everyone is willing to switch distros to use PipeWire right now.

Reply:
> Socket and devices are not the same thing.

Understood, I meant if `--devices=all` should be removed as it is usually preferable to remove unecessary sockets or devices. It seems it was added for ALSA (according to the comment).

But since Tenacity wants all three sound servers to be available, both `--socket=pulseaudio` and `--device=all ` should be kept for now.

Reply:
you need it also for MIDI. And RAW ALSA devices.